aries-site/src/themes/aries.js

```diff
@@ -89,6 +89,10 @@ export const aries = deepMerge(hpe, {
           height: 100%;
           justify-content: center;
         }
+        // align onSelect checkbox to center of header cell
```

Comment:
Why isn't this in the HPE theme?

Reply:
we will be moving all of this over to the HPE theme soon as the dataTable theming is fluctuating less now. As designs were getting finalized we wanted to iron items out first before placing on theme branch.